src/utils/client.js

```diff
@@ -5,7 +5,7 @@ import { initInfo, initState, initFormsPromise } from './init';
 import { encodeGetParams } from './primitives';
 
 // Convenience boolean for urls - manually change for testing
-const isLocal = false;
+const isLocal = true;
```

Comment:
this should be false

Reply:
Aah! I missed changing this setting. Reverted it. I Will push after addressing other comments as well!